pydatastructs/miscellaneous_data_structures/tests/test_queue.py

```diff
@@ -1,8 +1,18 @@
 from pydatastructs.miscellaneous_data_structures import Queue
 from pydatastructs.utils.raises_util import raises
+from pydatastructs.utils.misc_util import _check_type
 
 def test_Queue():
+    q = Queue(implementation='array')
+    q1 = Queue()
+    assert type(q) is type(q1)
+    q2 = Queue(implementation='linked_list')
+    assert _check_type(q2,Queue) is True
+    assert raises(NotImplementedError, lambda: Queue(implementation=''))
```

Comment:
```suggestion
    q = Queue(implementation='array')
    assert _check_type(q, Queue) is True
    q2 = Queue(implementation='linked_list')
    assert _check_type(q2, Queue) is True
    assert raises(NotImplementedError, lambda: Queue(implementation=''))
```

Reply:
`assert type(q) is type(q1)` is to check if default implementation of Queue is ArrayQueue and not some other implementation.

Reply:
`assert _check_type(q, ArrayQueue) is True`
Let me try this one.